tests/test_dpo_trainer.py

```diff
@@ -1135,9 +1136,10 @@ def test_dpo_trainer_use_num_logits_to_keep(self):
 class DPOVisionTrainerTester(unittest.TestCase):
     @parameterized.expand(
         [
-            ["trl-internal-testing/tiny-random-idefics2"],
-            ["trl-internal-testing/tiny-random-paligemma"],
-            ["trl-internal-testing/tiny-random-llava-1.5"],
+            ("qgallouedec/tiny-Idefics2ForConditionalGeneration",),
+            # ("qgallouedec/tiny-PaliGemmaForConditionalGeneration",),
```

Comment:
paligemma doesn't have a chat template (not meant for chat). We've to find a way to test it. Dedicated method?